winit-android/src/lib.rs

```diff
@@ -170,16 +161,5 @@ pub mod activity {
     // feature enabled, so we avoid inlining it so that they're forced to view
     // it on the crate's own docs.rs page.
     #[doc(no_inline)]
-    #[cfg(android_platform)]
     pub use android_activity::*;
```

Comment:
After removing these empty wrapper structs, is this now `pub use android_activity as activity;`?

Reply:
Yes, but I wanted to keep the doc comment above.

Reply:
I don't remember, doc-comments don't get forwarded on `pub use x as y;` rename-reexports?